src/gui/vector/qgsattributesformproperties.cpp

```diff
@@ -548,6 +586,11 @@ void QgsAttributesFormProperties::loadAttributeSpecificEditor( QgsAttributesDnDT
         loadAttributeContainerEdit();
         break;
       }
+      case DnDTreeItemData::Action:
+      {
+        receiver->clearSelection();
```

Comment:
why not calling selectFirstMatchingItem here?

Reply:
done